spec/factories/service_requests.rb

```diff
@@ -19,6 +22,7 @@
         'description' => description,
         'status' => status,
         'requested_datetime' => requested_datetime.iso8601,
+        'geometry' => geometry,
```

Comment:
The raw_data block should represent only what the Open311 api returns. This should be lat and lon. Then the subsequent test will be correct because it will test that 
the raw_data setter assigns the geometry correctly.

Reply:
This still needs to be resolved so it matches what would come back from an open311 api endpoint. It should be instead:

```ruby
raw_data do
  {
    # ...
    'lat' => lat,
    'long' => long,
  }
end
```

Reply:
Ah! I think I misunderstood how `raw_data` works. Hopefully good now.